Scanning dependencies of target scratch_wifi-experiment
[  0%] Building CXX object scratch/CMakeFiles/scratch_wifi-experiment.dir/wifi-experiment.cc.o
make[3]: *** [scratch/CMakeFiles/scratch_wifi-experiment.dir/build.make:65: scratch/CMakeFiles/scratch_wifi-experiment.dir/wifi-experiment.cc.o] Interrupt
make[2]: *** [CMakeFiles/Makefile2:17655: scratch/CMakeFiles/scratch_wifi-experiment.dir/all] Interrupt
make[1]: *** [CMakeFiles/Makefile2:17661: scratch/CMakeFiles/scratch_wifi-experiment.dir/rule] Interrupt
make: *** [Makefile:5844: scratch_wifi-experiment] Interrupt
Traceback (most recent call last):
  File "/usr/lib/python3.8/subprocess.py", line 1083, in wait
    return self._wait(timeout=timeout)
  File "/usr/lib/python3.8/subprocess.py", line 1806, in _wait
    (pid, sts) = self._try_wait(0)
  File "/usr/lib/python3.8/subprocess.py", line 1764, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/subprocess.py", line 1095, in wait
    try:
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mfaatz2s/ns-allinone-3.39/ns-3.39/ns3", line 1615, in <module>
    main()
  File "/home/mfaatz2s/ns-allinone-3.39/ns-3.39/ns3", line 1586, in main
    build_step(args,
  File "/home/mfaatz2s/ns-allinone-3.39/ns-3.39/ns3", line 1096, in build_step
    cmake_build(current_cmake_cache_folder,
  File "/home/mfaatz2s/ns-allinone-3.39/ns-3.39/ns3", line 895, in cmake_build
    ret = subprocess.run(cmake_build_command.split(),
  File "/usr/lib/python3.8/subprocess.py", line 495, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/lib/python3.8/subprocess.py", line 1020, in communicate
    self.wait()
  File "/usr/lib/python3.8/subprocess.py", line 1097, in wait
    except TimeoutExpired:
KeyboardInterrupt
